charms/kfp-metadata-writer/requirements.in

```diff
@@ -4,4 +4,7 @@
 charmed-kubeflow-chisme >= 0.2.0
 lightkube
 ops
+# pydantic greater or equal to 2.7 yields issues with the
+# rustc version available for this ubuntu version
+pydantic==2.6.4
```

Comment:
Can we specify the ubuntu version this applies to (20.04?)?  Is there something that'll get fixed that lets us remove this restriction, or an upstream bug to track?

Reply:
I'll add more information about the issue and when this pin can be removed.

Reply:
Updated, wdyt?

Reply:
lgtm! ty